docs/installation/configuration.rst

```diff
@@ -0,0 +1,101 @@
+Configuration
+=============
+
+.. _configuration:
+
+-----------
+
+Pulp uses `dynaconf <https://dynaconf.readthedocs.io/en/latest/>`_ for its settings. dynaconf
+allows storing `settings in multiple file formats <https://dynaconf.readthedocs
+.io/en/latest/guides/examples.html>`_. The location of the settings file is specified by setting
+the ``PULP_SETTINGS`` environment variable. Each of the settings can also be set by prepending
```

Comment:
I feel like there should be a default settings file location so a user won't necessarily have to muck with env vars if they don't want to?

Reply:
I file a RFE with dynaconf for this feature. https://github.com/rochacbruno/dynaconf/issues/87